packages/common/test/utils.spec.ts

```diff
@@ -57,122 +55,24 @@ describe('[Utils/Parse]', () => {
     assert.equal(params.genesis.timestamp, '0x10', 'timestamp parsed correctly')
   })
 
-  it('should successfully parse kiln genesis and set forkhash', async () => {
-    const common = createCommonFromGethGenesis(gethGenesisKilnJSON, {
```

Comment:
I don't think this one should be removed as well? Correctly setting the hardfork order? Maybe the `mergeForkIdTransition` should be removed here?

Reply:
I tried to re-add the two tests you mentioned, but I am getting a "Error parsing parameters file: nonzero terminal total difficulty is not supported" error now when trying to re-add the Kiln config file, so I guess this is not so easy to re-create in this setup.

I would generally pledge that there are more simple test setups for HF order as well as `getHardforkBy()` functionality and this is sufficiently tested in other places and we therefore shouldn't stick to this super-outdated Kiln setup (I was somewhat happy to throw the Kiln config out) and just throw it out here.

(side mention: guess nothing urgent to ad hoc re-add, but I do wonder if we would rather want to keep the "Geth config with post-Merge transition but a dedicated Merge HF block included" case (so this would also match what we keep in other places) instead of fully throwing this out. Guess *THAT* case also should not produce significant overhead and still migth be useful. Again, but in doubt we can "re-add on demand" I guess).